packages/core/src/components/forms/_controls.scss

```diff
@@ -354,7 +354,7 @@ $control-indicator-spacing: $pt-grid-size !default;
       transition: background-color $pt-transition-duration $pt-transition-ease;
 
       &::before {
-        position: relative;
+        position: absolute;
         left: 0;
```

Comment:
yeah, how was `left: 0` even supposed to do anything when `position: relative`

Reply:
it was more about the `left` when `:checked` but yea the issue here was obvious